[docs] Document #19259: Add a note in xCluster docs for ADD COLUMN ... DEFAULT (#20522)
yugabyte/yugabyte-db: docs/content/preview/deploy/multi-dc/async-replication/async-deployment.md

    ```output
    Replication enabled successfully
    ```
#### Adding a column with a non-volatile default value

> When adding a new column with a (non-volatile) default expression, make sure to perform the schema modification on the target with the **computed** default value. 

Reviewer: ```suggestion
When adding a new column with a (non-volatile) default expression, make sure to perform the schema modification on the target with the _computed_ default value. 
```

> Example:

Reviewer: ```suggestion
For example, say you have a replicated table `test_table`.
```

> Let's say we have a replicated table `test_table`.

Reviewer: ```suggestion

```

> 1. Pause replication

Reviewer: ```suggestion
1. Pause replication on both sides. 
```

Reviewer: Not sure about this suggestion. Please modify as needed.

1. Execute add column command on the source:
   ```sql
   ALTER TABLE test_table ADD COLUMN test_column TIMESTAMP DEFAULT NOW()
   ```

> 1. Perform the above `ALTER TABLE` command with the computed default value on the target:

Reviewer: ```suggestion
1. Run the preceding `ALTER TABLE` command with the computed default value on the target as follows:
```

>     - The computed default value can be retrieved from `pg_attribute.attmissingval`.

Reviewer: what is "pg_attribute.attmissingval". Maybe some detail about it can be helpful?

Author: how about something like this:

The computed default value can be retrieved from the `attmissingval` column in the `pg_attribute` catalog table.
